vowpalwabbit/cb_adf.cc

```diff
@@ -261,6 +263,11 @@ void cb_adf::learn_MTR(multi_learner& base, multi_ex& examples)
   examples[_gen_cs.mtr_example]->num_features = nf;
   examples[_gen_cs.mtr_example]->weight = old_weight;
   std::swap(_gen_cs.mtr_ec_seq[0]->pred.a_s, _a_s_mtr_cs);
+  if (_ccb)
```

Comment:
It is illegal for this reduction to set a label other than its specified output label type.

Reply:
Yes I just had this here to see the perf impact, I removed all changes to cb_adf in this PR